### DISCUSSION TITLE
Idea for the transport and call of the pokemons

### DISCUSSION THREAD

**Discussion by hitmanssilent** [Pokecube-Development/Pokecube-Issues-and-Wiki#828] [Ideas]
I can think of something like that the player could create a bag or a package in which we can store the pokecubes up to a certain limit, and with a simple right click the first pokecube that was from left to right would be called, if it were given a second click would call the second pokecubo from left to right, and when we press the "r" key to call them back, the pokecube looks for the previous space available inside the package or the bag from right to left, it also occurs to me that if keep the right click for a couple of seconds all the pokecubes inside the package or bag would be called at the same time, the same as if we hold down the "r" key for a few seconds so that they all return to their corresponding place searching from the last call up to the first one to place them from right to left (even if only a part of the pokemons has been called).
Also, if you wanted to call a specific pokemon only, it would be a matter of holding the bag or package in your main hand and pressing the assigned button to see the next or previous pokemon inside the bag or package and giving a single right click to call him.

Also, it occurs to me that I could configure this bag or package so that the player may or may not do the following things:
-Have more than one package or bag in your inventory
-Only being able to store pokemons inside that package or bag (and all the others that are caught are sent to the PC)
-Set the press and hold to call them all or call them back

This comes as an idea as a result of trying to put pokecubes in the "tool belt" mod but it did not accept the pokecube to be able to select and call them, so it occurred to me that this small problem could be solved with something like a bag or a package inside the mod or even some kind of compatibility with the tool belt mod could be added so that pokecubes of captured pokemons can be stored in the belt spaces.

**Comment by Thutmose** [comment]
So related to this issue here: <https://github.com/Pokecube-Development/Pokecube-Issues-and-Wiki/issues/859> I added an inventory for the belt wearables, that can hold some pokecubes. These currently are setup as not easy to access (need to press hotkey to open gui, then take out of the slot manually). An addon could be made to make the hotkeys as you suggest work for this belt.

I don't want to add that for the main functionaility of the belt, as I would rather keep the inventory "cost" of the pokecubes, ie they use the same inventory slots as regular tools, but it should be fairly simple to add in this feature via an addon.

**Reply by hitmanssilent** [reply]
Oh great, I already tried it and not only is it very functional but it also looks good aesthetically, with the pokecubes on the belt, I love it, thank you very much for adding this. As for the addon... I wish I knew how to program addons to add a feature like this, but I guess since I don't know, this will have to suffice, thank you very much for listening to my suggestion.
